publishing/ui-metadata/README.md

```diff
@@ -0,0 +1,51 @@
+## UI metadata
+
+By convention, a blendable web app must provide a metadata JSON that responds to the `app-name` blending token.
```

Comment:
Are you saying that the explanation or the implementation should be improved?

Improvements to the implementation are discussed here: https://github.com/Starcounter/RebelsLounge/issues/126. TL;DR `app-name` token will be changed to `app`, `icon` contexts.